package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.111.1",
+  "version": "2.111.2",
```

Comment:
Nit: It doesn't _really_ matter much, but technically according to [semver](https://semver.org/) this should be a minor version bump since it introduces new, backwards compatible, functionality.

Reply:
yeah I guess it should be considered new functionality, it was between a bug fix and new functionality to me

Reply:
Oh nice thanks for explaining this is good to know for the future